hackathon_site/dashboard/frontend/src/components/cart/CartErrorBox/index.tsx

```diff
@@ -3,10 +3,25 @@ import { useSelector } from "react-redux";
 import { cartTotalSelector, errorSelector } from "slices/hardware/cartSlice";
 import Grid from "@material-ui/core/Grid";
 import { Alert, AlertTitle } from "@material-ui/lab";
+import { teamSizeSelector } from "../../../slices/event/teamSlice";
```

Comment:
Absolute imports! It should be 
`"slices/event/teamSlice"` if you want pycharm to do this automatically go to Settings -> Editor -> Code Style -> JavaScript and you should be able to uncheck relative path imports